ssntp/server.go

```diff
@@ -223,6 +223,13 @@ func (server *Server) getSession(uuid string) *session {
 	return session
 }
 
+// SetClusterConfig recieves a payload previously created
+// in the config package and its applied in the
+// cluster configuration of the SSNTP server
+func (server *Server) SetClusterConfig(config []byte) {
```

Comment:
As explained above, the server.configuration.setConfiguration() should be called from server.Serve(), based on the configuration URI that we got from the server Config structure.
We only pass that information from a SSNTP server implementation down to ssntp/server.go, we do not call into neither an SSNTP configuration or a configuration API from a SSNTP server implementation.